releasenotes/notes/ground_state_interface-42576cb6658a46e0.yaml

```diff
@@ -10,12 +10,12 @@ features:
     Introduces ``chemistry/results`` where the eigenstate_result and the
     electronic_structure_result are also used for the algorithms.
     Introduces Minimum Eigensolver factories ``minimum_eigensolver_factories`` where chemistry specific
-    minimum eigensolvers can be initialised
+    minimum eigensolvers can be initialized
     Introduces orbital optimization vqe ``oovqe`` as a ground state solver for chemistry applications
 deprecations:
   - |
     ``Core Hamiltonian`` class is deprecated in favor of the ``FermionicTransformation``
-    ``Chemistry Operator`` class is deprecated in facor of the ``tranformations``
-    ``minimum_eigen_solvers/vqe_adapt`` is also deprecate and moved as an implementation
+    ``Chemistry Operator`` class is deprecated in favor of the ``tranformations``
+    ``minimum_eigen_solvers/vqe_adapt`` is also deprecated and moved as an implementation
```

Comment:
@woodsp-ibm as you can see the typos in the groundstate reno are now fixed in this PR. Sorry for the confusion in the other one.

Reply:
Ok, we'll sort it if there are conflicts then